app/src/main/java/com/babylon/wallet/android/presentation/settings/SettingsItem.kt

```diff
@@ -66,40 +113,48 @@ sealed interface SettingsItem {
         @DrawableRes
         fun getIcon(): Int? { // add rest of icons
             return when (this) {
-                SeedPhrases -> com.babylon.wallet.android.designsystem.R.drawable.ic_seed_phrases
-                LedgerHardwareWallets -> com.babylon.wallet.android.designsystem.R.drawable.ic_ledger_hardware_wallets
-                DepositGuarantees -> com.babylon.wallet.android.designsystem.R.drawable.ic_filter_list
                 is Backups -> com.babylon.wallet.android.designsystem.R.drawable.ic_backup
                 ImportFromLegacyWallet -> com.babylon.wallet.android.designsystem.R.drawable.ic_app_settings
                 AccountRecovery -> com.babylon.wallet.android.designsystem.R.drawable.ic_app_settings
             }
         }
     }
 
-    sealed interface AppSettingsItem {
-        data object LinkedConnectors : AppSettingsItem
-        data object Gateways : AppSettingsItem
-        data object EntityHiding : AppSettingsItem
-        data class DeveloperMode(val enabled: Boolean) : AppSettingsItem
-        data class CrashReporting(val enabled: Boolean) : AppSettingsItem
+    sealed interface WalletPreferencesSettingsItem {
```

Comment:
in this file I would also ditch the suffix `*SettingsItem` for those setting categories, since they are already under the namespace `SettingsItem`. 

So in the end I would have something like this:

`SettingsItem.WalletPreferences.CrashReporting`

instead of this

`SettingsItem.WalletPreferencesSettingsItem.CrashReporting`

if @sergiupuhalschi-rdx and @micbakos-rdx also agree